Documentation: Added sample output to output.md and Quick Start (#305)
nextflow-io/nf-co2footprint: README.md

More details are available in the Nextflow [plugin documentation](https://www.nextflow.io/docs/latest/plugins.html#plugins) and the [configuration guide](https://www.nextflow.io/docs/latest/config.html). 

> ### Output

Reviewer: ```suggestion
## Output
```
Otherwise it looks inconsistent or was it intended? 
In general I am wondering if we need this section here at all as we have it already in the Output section, what do you think?

Author: This was intended to be a subheading for the previous Quick-Start. But I don't care either way.

More details are available in the Nextflow [plugin documentation](https://www.nextflow.io/docs/latest/plugins.html#plugins) and the [configuration guide](https://www.nextflow.io/docs/latest/config.html). 

### Output

> The plugin generates three files, a [detailed report](assets/co2footprint_report_sample.html), a [text summary](assets/co2footprint_report_sample.html), and a [trace file](assets/co2footprint_report_sample.html) for more information have a look at the [output](usage/output.md) section.

Reviewer: ```suggestion
The plugin generates three files, a [detailed report](assets/co2footprint_report_sample.html), a [text summary](assets/co2footprint_report_sample.html), and a [trace file](assets/co2footprint_report_sample.html).  
For more details, see the [output documentation](usage/output.md).
```
